### DISCUSSION TITLE
"HitHighlightedSummary" property is not returning the searched keywords with <c0> tag, else it returns only plain text in PNP modern Search v4.9.3.0

### DISCUSSION THREAD

**Discussion by krishnare56** [microsoft-search/pnp-modern-search#3564] [Q&A]
I am using PNP modern search V4 webparts in SharePoint online site. where i have linked search box and search results webpart. i need to highlight the Searched keyword in search results, i referred some documents where it stats "HitHighlightedSummary" will return the summery with highlighting the matched keywords, but in my case its simply returning the summery without highlighting the keywords. i am not sure what i am missing.

i have tried adding "HitHighlightedSummary" in Hit-Highlighted properties to but result is still same.

please find the below image for reference, where i have searched with word GCS but in "HitHighlightedSummary" it still returns plain summary.
<img width="566" alt="0kKpl" src="https://github.com/microsoft-search/pnp-modern-search/assets/131133647/69131c7a-60a5-4da7-975f-1e91645681d8">

**Comment by kasperbolarsen** [comment]
Please update to the current version , 4.10.2 as we can't assist with older versions
